platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformWiring.java

```diff
@@ -871,7 +833,7 @@ public void bind(
         if (birthRoundMigrationShimWiring != null) {
             birthRoundMigrationShimWiring.bind(Objects.requireNonNull(birthRoundMigrationShim));
         }
-        latestCompleteStateNotifierWiring.bind(completeStateNotifier);
+        latestCompleteStateNotifierWiring.bind(builder.buildLatestCompleteStateNotifier());
```

Comment:
should this be `builder::buildLatestCompleteStateNotifier`?

Reply:
yes, fixed